frontend/tests/robot/RF-tests/resultpage/TestSuite.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Documentation    Tests for the frontpage of the app.
+
+Resource    suiteKeywords.resource
+Resource    ../testpages/suiteKeywords.resource
+Resource    ../../RF-keywords/CommonFunctions.resource
+
+Suite Setup
+...        Run Keywords
+...        Open And Configure Browser
+...        AND    Register Keyword To Run On Failure    Nothing
+Suite Teardown    Close Browser
+
+Test Setup    Go To Testpage
```

Comment:
test setuppi vois olla Go To Resultpage kosk täs suites testataa tulossivuu eikä testisivuja